### PR TITLE
Add article explaining how to provide cloud provider credentials via the Management API

### PR DESCRIPTION
**Pull request by marians**
Towards https://github.com/giantswarm/roadmap/issues/480

So far, only AWS is addressed.

### CODE REVIEW

**Comment by marians** [comment]
Dear people, I've almost rewritten the article after understanding better how the credentials handling works. Here is the outline of the current structure:

image

File: cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
+apiVersion: cluster.x-k8s.io/v1alpha3
```
Comment (axbarsan) [inline cluster.yaml:1]: I guess these 2 files have been committed by accident

Comment (marians) [reply cluster.yaml:1]: You're right! Thanks

Comment (marians) [reply cluster.yaml]: I removed them. FYI: The secret.yaml only contained dummy data.

**Pull request merged**